.gitignore

```diff
@@ -114,6 +114,5 @@ refix
 .kitchen
 .emacs.desktop*
 
-/test-suite.log
```

Comment:
this does not belong in this commit, please remove or make a new commit for just this change

Reply:
Removed